docs/en_US/autotune_ref.md

```diff
@@ -0,0 +1,80 @@
+# Python API Reference of Auto Tune
```

Comment:
This file can be an rst.

Reply:
just be consistent with NasReference.md, what is the reason of preferring rst?

Reply:
Can be. Not preferred to be. rst would save a few lines.